File: Managemate/ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Receipt_Models/Edit_Receipt_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Edit_Receipt_Data{
    session?: Session_Data;
    
    receipt_id:number;
    
    in_out : boolean;
    order_id_FK : number;
    element : string;
    date : Date;
    transport : string;
    comment : string;
}

File: Managemate/ManagemateAPI/ManagemateAPI/Management/M_Receipt/Input_Objects/Edit_Receipt_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Receipt.Input_Objects
{
    public class Edit_Receipt_Data
    {
        public Session_Data session { get; set; }

        public long receipt_id { get; set; }

        public string element { get; set; }

        public DateTime date { get; set; }

        public string transport { get; set; }

        public string comment { get; set; }

    }
}
